modules/nf-core/fastcov/meta.yml

```diff
@@ -0,0 +1,56 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: fastcov
+description: generate a coverage plot from one or more bam files
```

Comment:
Does this tool only work on bam files, or does it work on cram files too?

Reply:
Looking at the source code it seems to only works on bam files.